core/src/test/java/io/questdb/network/NetTest.java

```diff
@@ -73,6 +73,20 @@ public void testBindAndListenUdpToLocalhost() {
         }
     }
 
+    @Test
+    public void testGetAddrInfoConnect() {
```

Comment:
a test with an IP address instead of a host would be nice too. `getaddrinfo()` works with both IP address and a hostname. It saves a client a branch like this:
```java
if (looksLikeIP(stringAddress)) {
  oneConnectStrategy();
} else {
  anotherConnectStrategy();
}
```
Moreover when it's safe to assume (cloud) users will use hostname almost exclusively. (hence no reason to optimize for a fast-path with IP)

Reply:
i'll add test